packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -161,4 +177,43 @@ function onFetchParticipantMenuItems(): IContextualMenuItem[] {
   ];
 }
 
+const onFetchCustomButtonProps: CustomCallControlButtonCallback[] = [
+  (args: CustomCallControlButtonCallbackArgs): CustomCallControlButtonProps => {
+    return {
+      showLabel: args.displayType !== 'compact',
+      // Some non-default icon that is already registered by the composites.
+      onRenderOffIcon: () => <Icon iconName="MessageSeen" />,
```

Comment:
missing icons for some reason
![image](https://user-images.githubusercontent.com/2684369/149996975-abc3d4f9-5e19-42d7-8f22-0db1cd2d5e84.png)

Reply:
Automation hasn't updated the snapshots yet.

Reply:
![image](https://user-images.githubusercontent.com/82062616/150034397-71a4b9ba-7836-4f71-887c-e2e734bd228f.png)